README.md

```diff
@@ -8,12 +8,12 @@ We constantly aim to improve our products and processes, to both serve users' ne
 
 Liquid Collective has engaged independent security firms Halborn and Spearbit to perform security audits of the protocol. Every protocol feature deployed to mainnet has previously been reviewed by at least one of those teams.
 
-| **Date** | **Auditor** |                 **Scope**                 |                  **Link**                |
-|:--------:|:-----------:|:-----------------------------------------:|:----------------------------------------:|
-|  2023-03 |   Spearbit  | Withdrawals                               | [Spearbit #3](audits/202303_Spearbit.md) |
-|  2022-11 |   Spearbit  | TLC, Slashing Coverage, Protocol Metadata | [Spearbit #2](audits/202211_Spearbit.md) |
-|  2022-09 |   Spearbit  | Entire initial protocol                   | [Spearbit #1](audits/202209_Spearbit.md) |
-|  2022-07 |   Halborn   | Entire initial protocol                   | [Halborn #1](audits/202211_Spearbit.md)  |
+| **Date** | **Auditor** | **Security Researchers**                                                                                                                                                                                                                                                                                                                                                                                                               |                 **Scope**                 |                  **Link**                |
```

Comment:
Feels like it would be better for readibility to list auditors on each audit details page. So it does not make the summary heavier.